src/crewai/crew.py

```diff
@@ -295,6 +297,29 @@ def validate_end_with_at_most_one_async_task(self):
 
         return self
 
+    @model_validator(mode="after")
+    def validate_first_task(self) -> "Crew":
```

Comment:
Shouldn't we also validate that the last task isn't conditional?

Reply:
i would say last is fine no need to hard require. because the output will just be the previous output. something i can add a test for.